### PR TITLE
Add docs for email Astro alertrs

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/2489

### CODE REVIEW

File: static/astro-cli-release-notes.xml
Comment (jwitz) [inline static/astro-cli-release-notes.xml:1]: Safe to ignore these

File: astro/alerts.md [hunk 1]
```diff
@@ -75,6 +77,11 @@ To set up an alert integration with PagerDuty, you need access to your organizat
 
 5. Copy the **Integration Key** for your new Astro alert integration.
 
+</TabItem>
+<TabItem value="Email">
+
+No external configuration is required for the email integration.
```
[the remark this answers is not in the document]
Comment (AmandaYao00) [reply astro/alerts.md:83]: Good point! Could you recommend safelisting the domain astronomer.io?

**Review by AmandaYao00 — APPROVED** [review]

File: astro/alerts.md (hunk 1 above)
Comment (jwitz) [inline astro/alerts.md:83]:
```suggestion
No external configuration is required for the email integration. Astronomer recommends allowlisting `astronomer.io` with your email provider to ensure that no alerts go to your spam folder. Alerts are sent from `no-reply@astronomer.io`.
```

File: astro/alerts.md [hunk 2]
```diff
@@ -91,11 +98,18 @@ In the Cloud UI, you can enable alerts from the **Workspace Settings** page.
...
 
-5. Choose the **Communication Channel** where you want to send your alert. You can choose to send alerts to both Slack and PagerDuty.
+5. Choose the **Communication Channels** where you want to send your alert.
 
 6. Add your communication channel information.
 
-    <Tabs>
+    <Tabs
+        defaultValue="Slack"
+        groupId= "step-1-configure-your-communication-channel"
```
Comment (TJaniF) [inline astro/alerts.md:107]:
```suggestion
        groupId= "step-2-create-your-workspace-alert-in-the-cloud-ui"
```

Please ignore if this was left as the same groupId as in step 1 on purpose to have the tabs switch together :) I just faintly remember there was a reason to have it the same as the section title for linking?

**Review by TJaniF — APPROVED** [review]
LGTM!
I made two optional suggestions :) 
(also I assumed the files in the static folder are GH merge shenanigans and not part of this PR, if they are then please poke me to look at them as well :) )

File: astro/alerts.md [hunk 3]
```diff
@@ -91,11 +98,18 @@ In the Cloud UI, you can enable alerts from the **Workspace Settings** page.
...
 
-5. Choose the **Communication Channel** where you want to send your alert. You can choose to send alerts to both Slack and PagerDuty.
+5. Choose the **Communication Channels** where you want to send your alert.
 
 6. Add your communication channel information.
 
-    <Tabs>
+    <Tabs
+        defaultValue="Slack"
+        groupId= "step-2-create-your-workspace-alert-in-the-cloud-ui"
```
Comment (jwitz) [inline astro/alerts.md:107]:
```suggestion
        groupId= "step-1-configure-your-communication-channel"
```

File: astro/alerts.md [hunk 4]
```diff
@@ -9,26 +9,28 @@ import Tabs from '@theme/Tabs';
...
 ## Prerequisites
 
 - An [Astro project](develop-project.md).
 - An [Astro Deployment](create-deployment.md). Your Deployment must run Astro Runtime 7.1.0 or later to configure Astro alerts.
-- A Slack workspace and/or PagerDuty service.
+- A Slack workspace, PagerDuty service, or email address.
 
-## Step 1: Configure your communication channel
+
+## Step 1: Configure your communication channel <!-- Sensitive header used in product - do not change without a redirect-->
```
Comment (jwitz) [inline astro/alerts.md:25]:
```suggestion
## Step 1: Configure your communication channel
```

File: astro/alerts.md [hunk 5]
```diff
@@ -9,26 +9,28 @@ import Tabs from '@theme/Tabs';
...
 
 ## Prerequisites
 
 - An [Astro project](develop-project.md).
 - An [Astro Deployment](create-deployment.md). Your Deployment must run Astro Runtime 7.1.0 or later to configure Astro alerts.
-- A Slack workspace and/or PagerDuty service.
+- A Slack workspace, PagerDuty service, or email address.
 
-## Step 1: Configure your communication channel
+
```
Comment (jwitz) [inline astro/alerts.md:24]:
```suggestion
<!-- Sensitive header used in product - do not change without a redirect-->
```

**Pull request merged**